dev/src/bulk-writer.ts

```diff
@@ -372,6 +374,20 @@ export class BulkWriter {
     }
   }
 
+  /**
+   * Returns a lazy-loaded BulkWriter instance.
+   *
+   * @param firestore Used to instantiate the BulkWriter instance the first
+   * time.
+   * @private
+   */
+  static _getInstance(firestore: Firestore): BulkWriter {
+    if (!this.instance || this.instance.firestore !== firestore) {
```

Comment:
I can see arguments either way, but at this point, I'm more in favor of leaving it on the Firestore instance, since trying to track the different firestore instances to pass into BulkWriter is more confusing than just having the BulkWriter instance available on the Firestore instance.

Reverted back to before.